std/typecons.d

```diff
@@ -2508,7 +2508,10 @@ Forces $(D this) to the null state.
  */
     void nullify()()
     {
-        .destroy(_value);
+        static if (is(T == class))
```

Comment:
Should this be `if (is(T == class) || is(T == interface))` ?

Reply:
You're right, I missed that.  Fixed.